projects/igniteui-angular/migrations/update-11_1_0/index.spec.ts

```diff
@@ -708,47 +708,4 @@ export class CsvExportComponent {
             )
         ).toEqual(expectedContent);
     });
-
-    it('should update GridPagingMode enum from lowerCase to TitleCase', async () => {
```

Comment:
Assuming these don't work because the resolution is based on the TypeScript LS checking `GridPagingMode` and verifying it's an igniteui-angular export, the member is based pure string match. Perhaps we have a class declaration or enum filter that fails the check for the new const. Might want to check, even though not really critical for this PR